indico/modules/events/surveys/fields/choices.py

```diff
@@ -29,6 +30,23 @@
 from indico.web.forms.validators import HiddenUnless
 
 
+class OptionIDMixin(object):
+
+    def copy_field_data(self):
+        """Return a copy of the field's configuration data without the IDs used to identify answer options."""
+        field_data_copy = super(OptionIDMixin, self).copy_field_data()
+        for option in field_data_copy['options']:
+            del option['id']
+        return field_data_copy
+
+    @staticmethod
+    def process_imported_data(form):
+        """Generate the options' IDs"""
+        if 'options' in form.data:
```

Comment:
Modifying this is not a good idea. `form.data` is generated on the fly when you access that property and I don't think it's a good idea to actually modify the form data outside the form at all.